docs/resources/service-providers.md

```diff
@@ -4,49 +4,161 @@ order: 5
 
 # Service Providers
 
-We define 'service providers' as entities providing services for end-users that involve some form of interaction with a Cosmos-SDK based blockchain (this includes the Cosmos Hub). More specifically, this document will be focused around interactions with tokens.
+'Service Providers' are defined as entities providing services for end-users that involve some form of interaction with the Cosmos Hub. More specifically, this document will be focused around interactions with tokens.
 
-This section does not concern wallet builders that want to provide Light-Client functionalities. Service providers are expected to act as trusted point of contact to the blockchain for their end-users. 
+This section does not concern wallet builders that want to provide Light-Client functionalities. Service Providers are expected to act as trusted point of contact to the blockchain for their end-users.
 
-## High-level description of the architecture
+This document describes 
 
-There are three main pieces to consider:
+- [Connection Options](#connection-options)
+- [Running a Full Node](#running-a-full-node)
+  - [What is a Full Node?](#what-is-a-full-node)
+  - [Installation and Configuration](#installation-and-configuration)
+- [Command-Line Interface](#command-line-interface)
+  - [Available Commands](#available-commands)
+  - [Remote Access to gaiad](#remote-access-to-gaiad)
+  - [Create a Key pair](#create-a-key-pair)
+  - [Check your Account](#check-your-account)
+  - [Check your Balance](#check-your-balance)
+  - [Send coins using the CLI](#send-coins-using-the-cli)
+- [REST API](#rest-api)
+  - [Listen for incoming transactions](#listen-for-incoming-transaction)
 
-- Full-nodes: To interact with the blockchain. 
-- Rest Server: This acts as a relayer for HTTP calls.
-- Rest API: Define available endpoints for the Rest Server.
+
+## Connection Options
+
+There are four main technologies to consider, connecting to the Cosmos Hub:
+
+- Full Nodes: To interact with the blockchain. 
+- REST Server: This acts as a relayer for HTTP calls.
+- REST API: Define available endpoints for the REST Server.
+- GRPC: Connect to the Cosmos Hub via gRPC.
 
 ## Running a Full-Node
 
+### What is a Full Node?
+
+A Full Node is a network node that syncs up with the state of the blockchain. It provides blockchain data to others by using RESTful APIs, a replica of the database by exposing data with interfaces. A Full Node keeps in syncs with the rest of the blockchain nodes and stores the state on disk. If the full node does not have the queried block on disk the full node can go find the blockchain where the queried data lives. 
```

Comment:
```suggestion
A full node is a network node that syncs up with the state of the blockchain. A full node provides blockchain data to other blockchains by using RESTful APIs. A full node is a replica of the database by exposing data with interfaces. A full node keeps in syncs with the rest of the blockchain nodes and stores the state on disk. If the full node does not have the queried block on disk, the full node can go find the blockchain where the queried data lives. 
```

Reply:
I am learning! what do we mean by? (is a full node a replica?)

> by using RESTful APIs. A full node is a replica of the database by exposing data with interfaces.

Reply:
`to other blockchains` is not correct. It providers blockchain data to other `nodes` using `the same blockchain`.